client/my-sites/promote-post-i2/utils/index.ts

```diff
@@ -172,6 +188,12 @@ export const formatNumber = ( number: number, onlyPositives = false ): string =>
 	return number.toLocaleString();
 };
 
+export const canPromoteCampaignAgain = ( status: string ) => {
+	return [ campaignStatus.FINISHED, campaignStatus.CANCELED, campaignStatus.REJECTED ].includes(
```

Comment:
Is this check necessary? I think the user can promote again as much as he wants, no matter the status of this specific campaign. 

In general, you can have multiple campaigns running simultaneously for the same post.